src/main/scala/scorex/core/validation/ModifierValidator.scala

```diff
@@ -26,7 +26,7 @@ import scala.util.Try
 trait ModifierValidator {
 
   /** Encoder from bytes to string and back for logging */
-  implicit val encoder: BytesEncoder
+  implicit val encoder: ScorexEncoder
```

Comment:
i'd like to drop `encoder` here and use just `id.toString` method as `id` is not an array of bytes anymore